webapp/channels/src/components/drafts/draft_row.tsx

```diff
@@ -373,40 +373,51 @@ function DraftRow({
             innerRef={scrollIntoView ? alertRef : undefined}
             isHighlighted={scrollIntoView}
             className={containerClassName}
+            ariaLabel={intl.formatMessage({
```

Comment:
Sorry for keeping getting you to change this, but could you change the outer message to include the "scheduled post" and "draft" part? Ignoring the "deleted channel" case, that that would mean we only have 1 strings to translate instead of 3